robot/framesystem/framesystem.go

```diff
@@ -328,7 +328,10 @@ func (svc *frameSystemService) updateLocalParts(ctx context.Context) error {
 		seen[c.Name] = true
 		model, err := extractModelFrameJSON(svc.r, c.ResourceName())
 		if err != nil && !errors.Is(err, referenceframe.ErrNoModelInformation) {
-			return err
+			// When we have non-nil erros here, it is because the resource is not yet available.
+			// In this case, we will exclude it from the FS.
+			// When it becomes available, it will be included.
+			continue
```

Comment:
This feels like an improvement but would be a pretty big change from current behavior, right?

Reply:
It's not actually a behavior change in terms of meaningful results. Without this change, the FS service spouts errors until the resource is available, at which point everything is fine. This way, those errors are silenced, but because the only way for this to happen is for a resource to be in the resource graph but error on actual retrieval, conditions for errors here only occur briefly during startup.

Making the FS service tolerant of not-yet-initialized resources was Eric's suggested solution here.

The only behavioral difference this causes is the suppression of these spurious error messages. Both before and now, an attempt to use an uninitialized resource will fail. Manual testing with remotes and such yielded no issues.

Reply:
Shorter answer, this changes no actual *behavior*, will just mildly change which error message a user will get in certain scenarios.

Reply:
Sounds great, I'm all for this

Reply:
I think the worst case scenario is that a component actually doesn't have ModelFrame information but is skipped and the user isn't aware until they try to move the component. I feel like we should log a warning for this but continue with `continue`

Reply:
Anything that would normally have model frame information but does not, should error all on its own during its own initialization.